client/webserver/site/src/js/wallets.ts

```diff
@@ -346,58 +257,255 @@ export default class WalletsPage extends BasePage {
+      Doc.empty(tmpl.baseSymbol, tmpl.quoteSymbol)
+      tmpl.baseSymbol.appendChild(Doc.symbolize(basesymbol))
+      tmpl.quoteSymbol.appendChild(Doc.symbolize(quotesymbol))
+
+      if (spot) {
+        const convRate = app().conventionalRate(baseid, quoteid, spot.rate)
+        tmpl.price.textContent = fourSigFigs(convRate)
+        tmpl.priceQuoteUnit.textContent = quotesymbol.toUpperCase()
+        tmpl.priceBaseUnit.textContent = basesymbol.toUpperCase()
+        tmpl.volume.textContent = fourSigFigs(spotVolume(assetID, mkt))
+        tmpl.volumeUnit.textContent = assetID === baseid ? basesymbol.toUpperCase() : quotesymbol.toUpperCase()
+      } else Doc.hide(tmpl.priceBox, tmpl.volumeBox)
+      Doc.bind(row, 'click', () => app().loadPage('markets', { host, base: baseid, quote: quoteid }))
+    }
+    page.marketsOverviewBox.classList.remove('invisible')
+  }
+
+  async showRecentActivity (assetID: number) {
+    const page = this.page
+    // const loaded = app().loading(this.main)
+    const filter: OrderFilter = {
+      n: 20,
+      assets: [assetID],
+      hosts: [],
+      statuses: []
+    }
+    const res = await postJSON('/api/orders', filter)
+    // loaded()
+    Doc.hide(page.noActivity, page.orderActivity)
+    if (!res.orders || res.orders.length === 0) {
+      Doc.show(page.noActivity)
+      page.orderActivityBox.classList.remove('invisible')
+      return
+    }
+    Doc.show(page.orderActivity)
+    Doc.empty(page.recentOrders)
+    for (const ord of (res.orders as Order[])) {
+      const row = page.recentOrderTmpl.cloneNode(true) as PageElement
+      page.recentOrders.appendChild(row)
+      const tmpl = Doc.parseTemplate(row)
+      let from: string, to: string
+      const [baseUnitInfo, quoteUnitInfo] = [app().unitInfo(ord.baseID), app().unitInfo(ord.quoteID)]
+      if (ord.sell) {
+        [from, to] = [ord.baseSymbol, ord.quoteSymbol]
+        tmpl.fromQty.textContent = Doc.formatCoinValue(ord.qty, baseUnitInfo)
+        if (ord.type === OrderUtil.Limit) {
+          tmpl.toQty.textContent = Doc.formatCoinValue(ord.qty / OrderUtil.RateEncodingFactor * ord.rate, quoteUnitInfo)
+        }
+      } else {
+        [from, to] = [ord.quoteSymbol, ord.baseSymbol]
+        if (ord.type === OrderUtil.Market) {
+          tmpl.fromQty.textContent = Doc.formatCoinValue(ord.qty, baseUnitInfo)
+        } else {
+          tmpl.fromQty.textContent = Doc.formatCoinValue(ord.qty / OrderUtil.RateEncodingFactor * ord.rate, quoteUnitInfo)
+          tmpl.toQty.textContent = Doc.formatCoinValue(ord.qty, baseUnitInfo)
+        }
+      }
+
+      tmpl.fromLogo.src = Doc.logoPath(from)
+      Doc.empty(tmpl.fromSymbol, tmpl.toSymbol)
+      tmpl.fromSymbol.appendChild(Doc.symbolize(from))
+      tmpl.toLogo.src = Doc.logoPath(to)
+      tmpl.toSymbol.appendChild(Doc.symbolize(to))
+      tmpl.status.textContent = OrderUtil.statusString(ord)
+      tmpl.filled.textContent = `${(ord.filled / ord.qty * 100).toFixed(1)}%`
```

Comment:
Need to use the new `OrdreUtil.filled` for this percent, from https://github.com/decred/dcrdex/pull/1744